paddle/phi/core/distributed/comm_task.h

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) 2023 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <chrono>
+#include <condition_variable>
+#include <cstdint>
+#include <exception>
+#include "glog/logging.h"
+#include "paddle/phi/core/distributed/utils.h"
+#include "paddle/phi/core/enforce.h"
+#include "paddle/phi/core/macros.h"
+
+namespace phi {
+namespace distributed {
+
+class Store;
+class CommTask {
+ public:
+  CommTask(const std::string& backend = "",
+           const phi::Place& place = phi::Place(),
+           int rank = -1,
+           int size = 0,
+           int gid = 0,
+           uint64_t seq = 0,
+           int64_t numel = 0,
+           CommType comm_type = CommType::UNKNOWN)
+      : backend_(backend),
+        place_(place),
+        rank_(rank),
+        size_(size),
+        gid_(gid),
+        seq_(seq),
+        numel_(numel),
+        comm_type_(comm_type) {
+    const char* global_rank = std::getenv("PADDLE_TRAINER_ID");
+    PADDLE_ENFORCE_NOT_NULL(
+        global_rank,
+        phi::errors::NotFound(
+            "The environment variable 'PADDLE_TRAINER_ID' cannot be found."));
+    global_rank_ = std::atoi(global_rank);
+  }
+  virtual ~CommTask() = default;
+
+  std::string GetBackend() { return backend_; }
```

Comment:
```suggestion
  std::string backend() { return backend_; }
```
It is simpler, the same for other `get` function.